pageserver/src/layered_repository/timeline.rs

```diff
@@ -205,6 +205,8 @@ pub struct LayeredTimeline {
     pub layers: RwLock<LayerMap>,
 
     last_freeze_at: AtomicLsn,
+    // Atomic would be more appropriate here.
```

Comment:
I thought about that and there is even a ready snippet
https://stackoverflow.com/a/42563246/4014587
but it disappoints me with feeling like this is a hack. Timestamp is likely internally is the same u64, it would be more straightforward just access it directly through atomic operations...